README_RECON.md

```diff
@@ -0,0 +1,165 @@
+# Remorph Reconciliation
+
+Remorph Reconciliation is a library that helps to reconcile the data, post data migration or during the data migration   
+between the source system and databricks system.It supports multiple types of reconciliation depending upon the user   
+requirements and data in the source system.It makes the migration process simpler by removing the complex part of
+validating the data from one system to another and enabling the team with more insights on the data migrated.
+
+## Installation
+
+## Flow Chart for supported Report Type and Reconciliation Logics
+
+```mermaid
+flowchart TD
+    REPORT_TYPE --> DATA
+    REPORT_TYPE --> SCHEMA
+    REPORT_TYPE --> ROW
+    REPORT_TYPE --> ALL
+```
+
+```mermaid
+flowchart TD
+    SCHEMA --> SCHEMA_VALIDATION
+```
+
+```mermaid
+flowchart TD
+    ROW --> MISSING_IN_SRC
+    ROW --> MISSING_IN_TGT
+```
+
+```mermaid
+flowchart TD
+   DATA --> MISMATCH_ROWS
+   DATA --> MISSING_IN_SRC
+   DATA --> MISSING_IN_TGT
+```
+
+```mermaid
+flowchart TD
+    ALL --> MISMATCH_ROWS
+    ALL --> MISSING_IN_SRC
+    ALL --> MISSING_IN_TGT
+    ALL --> SCHEMA_VALIDATION
+```
+
+## Supported Source System and Report Types
+
+| Source     | Schema | Row | Data | All |
+|------------|--------|-----|------|-----|
+| Oracle     | Yes    | Yes | Yes  | Yes |
+| Snowflake  | Yes    | Yes | Yes  | Yes |
+| Databricks | Yes    | Yes | Yes  | Yes |
+
+## Dependency CLI commands
+
+TBD
```

Comment:
CLI will trigger Jobs, @bishwajit-db or @vijaypavann-db who is working on the cli piece?